docassemble/juvenilesealing/data/questions/document-download-screen.yml

```diff
@@ -110,37 +82,43 @@ fields:
   - note: ${ action_button_html("#", id_tag="sendAttachmentEmail", classname="atch-button", new_window=atch_do_new_window_email, icon="envelope", label=atch_send_email_label, color="primary", size="md") }
 #  - note: ${ action_button_html("#", id_tag="downloadAttachment", classname="atch-button", new_window=atch_do_new_window_download, icon="download", label=atch_download_label, color="primary", size="md") }
   - note: ${atch_after_download}
+
 script: |
   <script>
-    console.log(val('email_sent_prefix'))
-
+  
     $("#sendAttachmentEmail").click(function(){
-
-      let message = val('email_attempt_prefix') + val('atch_email_address') + ".";
+      daShowSpinner();
+      
+      var email_attempt_prefix = ${ json.dumps(str(email_attempt_prefix))}
+      var message = email_attempt_prefix + val('atch_email_address') + ".";
```

Comment:
This needs a space between the attempt_prefix text and the email address: `email_attempt_prefix + " " + val('atch_email_address') + "."`